.travis.yml

```diff
@@ -0,0 +1,47 @@
+--- 
+matrix: 
```

Comment:
* add mongodb as in the Travis CI test environment. Some scripts are using an actual instance for testing their logic out.

Reply:
@rajatkb I have added mongodb as service. Is it ok now?